src/dut.rs

```diff
@@ -417,7 +417,7 @@ impl SshInfo {
         // > Support for <zone_id> is intentionally omitted.
         // So that we can not parse dut string like:
         // > [fe80::8a54:1fff:fe0f:72a5%en0]
-        // TODO(hikalium): Impl workaround, possibly fix in the url crate.
+        // TODO(hikacro3): Impl workaround, possibly fix in the url crate.
```

Comment:
are you sure? 😄

Reply:
aaaaaaahhhh, good catch, thanks... (I'm not going to change my name to hikacro3 😂 )